pymic/offload_stream.py

```diff
@@ -562,6 +562,9 @@ def invoke(self, kernel, *args):
             raise OffloadError("Cannot invoke kernel, "
                                "library not loaded on device")
 
+        # Map None to nullptr
+        args = [arg if arg is not None else 0 for arg in args]
```

Comment:
That's correct.  The `0` value on the host is treated as a scalar and the kernel receives a pointer to it.  In theory. LIBXSTREAM supports passing a true `nullptr` to the kernel.